File: AtlasWeb/Core/wwwroot/app/NebulaClient/StorageManager/DefinitionManagement/TTListColumnDef.ts
import { Guid } from "../../Mscorlib/Guid";
import { TTListDef } from "./TTListDef";
import { TTObjectPropertyDef } from "./TTObjectPropertyDef";
import { TTObjectCodedPropertyDef } from "./TTObjectCodedPropertyDef";

export class TTListColumnDef {
    public ListDef: TTListDef;
    public ListDefID: Guid;
    public ListColumnDefID: Guid;
    public PropertyDef: TTObjectPropertyDef;
    public CodedPropertyDef: TTObjectCodedPropertyDef;
    public MemberName: string;
    public Caption: string;
    public ColumnOrder: number;
    public ColumnWidth: number;
    public ToString(): string {
        return null;
    }
}

File: ATLASDEV/AtlasModel/Enterprise/TTListColumnDef.cs
using System;

namespace AtlasModel.Enterprise
{
    public partial class TTListColumnDef
    {
        public Guid ListColumnDefId { get; set; }
        public Guid ListDefId { get; set; }
        public string MemberName { get; set; }
        public string Caption { get; set; }
        public byte ColumnOrder { get; set; }
        public int ColumnWidth { get; set; }
        public short CheckOutStatus { get; set; }
    }
}